final_task/rss_reader/converter.py

```diff
@@ -0,0 +1,62 @@
+import os
+import pdfkit
+from dominate.tags import div, h2, img, p, link
+from datetime import datetime
+
+import news_parser
+import logger
+
+
+def createHtmlStructure(channel, limit, html_document, html_path, pdf_path):
+    '''
+    1. in loop create html structure
+    2. create html document or convert html structure into pdf
+    '''
+    with html_document.head:
+        styles = os.path.abspath('style.css')
+        link(rel='stylesheet', href=styles)
+    for index, item in enumerate(channel.entries):
+        if (index == limit):
+            break
+        with html_document:
+            with div():
+                h2("Title: " + item.title)
+                p("Link: " + item.link)
+                media_content = news_parser.checkMediaContent(item)
+                if (media_content):
+                    img(src=media_content)
+                p("Description: " + news_parser.getDescription(item.description))
+
+    if (html_path):
+        intoHTML(html_document, html_path)
+    else:
+        intoPDF(html_document, pdf_path)
+
+
+def intoHTML(html_document, html_path):
+    '''
+    1. create folder with html file
+    2. write html structure in file
+    '''
+    if not os.path.exists(html_path):
+        os.makedirs(html_path)
+    time_name = datetime.strftime(datetime.now(), "%H%M%S")
+    html_file = html_path + '/' + time_name + '.html'
```

Comment:
It would be nice to make name more informative, for exaple:
```NewsFeed-<all/limited_to_<number>>-<datetime>.html```

Reply:
Fixed